### PR TITLE
THE FINAL WORKING COPY

### PR DESCRIPTION
**Pull request by dennislee15**
it works

### CODE REVIEW

File: src/main/java/conf/Routes.java [hunk 1]
```diff
@@ -30,8 +30,15 @@ public void init(Router router) {
         router.GET().route("/").with(ApplicationController.class, "index");
         router.GET().route("/hello_world.json").with(ApplicationController.class, "helloWorldJson");
         router.GET().route("/Blackjack").with(ApplicationController.class, "blackjack");
-        
- 
+        router.GET().route("/game/player").with(ApplicationController.class, "playerGameGet");
+        router.GET().route("/game/dealer").with(ApplicationController.class, "dealerGameGet");
+        router.POST().route("/hit/player/{columnTo}").with(ApplicationController.class, "playerHit");
+        router.POST().route("/split").with(ApplicationController.class, "split");
+        router.POST().route("/noSplitData").with(ApplicationController.class, "dealerPlay");
```
Comment (charan96) [inline src/main/java/conf/Routes.java:37]: what does this route do?

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -103,9 +104,33 @@ public int getTotalScore(ArrayList<String> scores)
         return total;
     }
 
-    public String checkWinner(Game dealer,Game player)
+    public void checkWinner(int dScore,int pScore, int spScore)
```
Comment (schultsa) [inline src/main/java/models/Game.java:107]: This was a needed addition.

File: src/main/java/conf/Routes.java (hunk 1 above)
Comment (dennislee15) [inline src/main/java/conf/Routes.java:37]: it does a lot of stuff

File: src/main/java/models/playerGame.java [hunk 3]
```diff
@@ -21,24 +21,31 @@ public void initialDeal()
     @Override
     public void hit(int colNum)
     {
-        if (playerScore >= 21)
-        {
-            return;
+        if(colNum ==1) {
```
Comment (schultsa) [inline src/main/java/models/playerGame.java:24]: Why do these just return and do nothing?

File: src/main/java/conf/Routes.java (hunk 1 above)
Comment (charan96) [inline src/main/java/conf/Routes.java:37]: i understand now

File: src/main/java/models/playerGame.java (hunk 3 above)
Comment (schultsa) [inline src/main/java/models/playerGame.java:24]: Ahh I see it is Bust conditions.

File: src/main/java/conf/Routes.java (hunk 1 above)
Comment (dennislee15) [inline src/main/java/conf/Routes.java:37]: it passes over the control to the dealer once the player hits stay

**Pull request merged**